.github/workflows/pip.yml

```diff
@@ -21,10 +21,28 @@ jobs:
           repository: llvm/llvm-project
           path: llvm-project
           # TODO: set the ref to match the major version of Halide
+      - uses: actions/checkout@v3
+        with:
+          repository: madler/zlib
+          path: zlib
+          ref: v1.2.12
       - name: Install dependencies
         run: |
           yum makecache
-          yum install -y ninja-build zlib-static
+          yum install -y ninja-build
+      - name: Configure ZLIB
+        run: >
+          cmake 
```

Comment:
...how the heck does a Linux Docker image not have zlib already installed

Reply:
It has shared zlib only, but we need static